contrib/rpm/buildah.spec

```diff
@@ -26,7 +26,7 @@
 
 Name:           buildah
 # Bump version in buildah.go too
-Version:        1.8-dev
```

Comment:
The version listed in buildah.go should match this.

Reply:
What I am proposing is that the '-dev' become 'dev' in the spec/rpm file only.

Below quote is from Max RPM which was published in 2000... so it's been this way for about 20 years.. probably more 😁 

RPM spec reference here:
http://ftp.rpm.org/max-rpm/ch-rpm-file-format.html
> The version number is normally taken verbatim from the package's version. The only restriction placed on the version is that it cannot contain a dash "-".

Reply:
Guess I was showing my rpm spec lack of knowledge off well!
I'm OK, but now wild with dev smashed up against the number.  Could we change it to 1.8.dev?  or is that no good either?

If possible, I'd like to keep the version the same in both this file and buildah.go.

Reply:
Yeah '.' should be good, and there is precedent for this in the RHEL repos, not with 'dev' specifically, but string content eg. commit id's.